app/package/controllers.py

```
@@ -187,10 +181,6 @@ def undelete_data_package(publisher, package):
         status_db = Package.change_status(publisher, package, PackageStateEnum.active)
         if status_acl and status_db:
             return jsonify({"status": "OK"}), 200
-        if not status_acl:
-            raise Exception('Failed to change acl')
-        if not status_db:
-            raise Exception('Failed to change status')
     except Exception as e:
```

Remove this try/except again - let the error propagate.